### DISCUSSION TITLE
First OSS Contribution!

### DISCUSSION THREAD

**Discussion by mariachrisochoou** [CatimaLoyalty/Android#1782] [Q&A]
Hey, I am very interested in this project and I was wondering if there is an issue I can help with as a first-time contributor. A little guidance would be very much appreciated. Thank you!

**Comment by TheLastProject** [comment]
Hi!

Normally those would be issues labelled [good first issue](https://github.com/CatimaLoyalty/Android/labels/good%20first%20issue) but looking at that list I fear it might have gotten fairly outdated.

I think #1506 might be a good first issue, it is about the app crashing if you for example put a huge amount of 9s in the balance field. I think you may want to look at every place you can fill in an amount (Edit -> Options -> Balance) and after setting a balance try if the little shopping cart icon in the screen where you look at a loyalty card also crashes with lots of 9s.

Of course, first steps first: install Android Studio and try to run the app :)
